### PR TITLE
Update latest details about external collaboration settings

### LINKED ISSUE #2007: Organizational Relationships Setting Moved
"Organizational relationships" has been moved to Dashboard &gt; External Identities &gt; External collaboration settings

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 147fb47a-99b2-f8ef-9ca6-35696d49c3fb
* Version Independent ID: b2f1b521-f231-48c2-b61f-f71bf9248d1c
* Content: [Collaborate with guests in a team](https://docs.microsoft.com/en-us/microsoft-365/solutions/collaborate-as-team?view=o365-worldwide#feedback)
* Content Source: [microsoft-365/solutions/collaborate-as-team.md](https://github.com/MicrosoftDocs/microsoft-365-docs/blob/public/microsoft-365/solutions/collaborate-as-team.md)
* Product: **microsoft-365-enterprise**
* GitHub Login: @MikePlumleyMSFT
* Microsoft Alias: **mikeplum**

### PR DESCRIPTION
**Pull request by PrasadKasireddy**
#### Category
- [x] Content fix
- [ ] New article
- [ ] Example checked item (*delete this line*)

#### Related issues
 - fixes https://github.com/MicrosoftDocs/microsoft-365-docs/issues/2007

### CODE REVIEW

**Comment by denisebmsft** [comment]
@MikePlumleyMSFT, you might need to fix the metadata section for formatting

**Comment by PrasadKasireddy** [comment]
> @MikePlumleyMSFT, you might need to fix the metadata section for formatting

Thanks for your inputs, I am new to this community, could you please guide me what meta data I need to update, I can see there is a meta data for this article on top of the page

image

**Comment by mypil** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: microsoft-365/solutions/collaborate-as-team.md [hunk 1]
```diff
@@ -29,25 +29,34 @@ This video shows the configuration steps described in this document.</br>
 
 > [!VIDEO https://www.microsoft.com/videoplayer/embed/RE44NTr?autoplay=false]
 
-## Azure Organizational relationships settings
+## To configure external collaboration settings:
```
Comment (JohanFreelancer9) [inline microsoft-365/solutions/collaborate-as-team.md]:
```suggestion
## Configuring external collaboration settings
```

File: microsoft-365/solutions/collaborate-as-team.md [hunk 2]
```diff
@@ -29,25 +29,34 @@ This video shows the configuration steps described in this document.</br>
...
 
-Sharing in Microsoft 365 is governed at its highest level by the organizational relationships settings in Azure Active Directory. If guest sharing is disabled or restricted in Azure AD, this will override any sharing settings that you configure in Microsoft 365.
+Sharing in Microsoft 365 is governed at its highest level by the external collaboration settings in Azure Active Directory. If guest sharing is disabled or restricted in Azure AD, this will override any sharing settings that you configure in Microsoft 365.
 
-Check the organizational relationships settings to ensure that sharing with guests is not blocked.
+To set external collaboration settings
```
Comment (JohanFreelancer9) [inline microsoft-365/solutions/collaborate-as-team.md]:
```suggestion
To set external collaboration settings:
```

File: microsoft-365/solutions/collaborate-as-team.md [hunk 3]
```diff
@@ -29,25 +29,34 @@ This video shows the configuration steps described in this document.</br>
...
+2. In the left navigation, click **Azure Active Directory** > **User settings**.
+3. Click **Manage external collaboration settings**.
+4. On the External collaboration settings page, choose the policies you want to enable.
 5. Ensure that **Admins and users in the guest inviter role can invite** and **Members can invite** are both set to **Yes**.
 6. If you made changes, click **Save**.
 
 Note the settings in the **Collaboration restrictions** section. Make sure that the domains of the guests that you want to collaborate with aren't blocked.
 
+  > [!NOTE]
+  > Also, you can set external collaboration settings another way, click **Azure Active Directory** > **External Identities** > **External collaboration settings**.
```
Comment (JohanFreelancer9) [inline microsoft-365/solutions/collaborate-as-team.md]:
```suggestion
  > You can also set external collaboration settings by selecting **Azure Active Directory** > **External Identities** > **External collaboration settings**.
```

File: microsoft-365/solutions/collaborate-as-team.md [hunk 4]
```diff
@@ -29,25 +29,34 @@ This video shows the configuration steps described in this document.</br>
...
+  
+  ![Screenshot of Azure Active Directory External Collaboration Settings page](../media/azure-ad-external-collaboration-settings.png)
+  
+  - **Guest users permissions are limited**: This policy determines permissions for guests in your directory. Select **Yes** to block guests from certain directory tasks, like enumerating users, groups, or other directory resources. Select **No** to give guests the same access to directory data as  regular users in your directory.
```
Comment (JohanFreelancer9) [inline microsoft-365/solutions/collaborate-as-team.md:52]:
```suggestion
  - **Guest users permissions are limited**: This policy determines permissions for guests in your directory. Select **Yes** to block guests from certain directory tasks, like enumerating users, groups, or other directory resources. Select **No** to give guests the same access to directory data as regular users in your directory.
```

Comment (JohanFreelancer9) [inline microsoft-365/solutions/collaborate-as-team.md]:
```suggestion
  - **Guest users permissions are limited**: This policy determines permissions for guests in your directory. Select **Yes** to block guests from certain directory tasks, like enumerating users, groups, or other directory resources. Select **No** to give guests the same access to directory data as regular users in your directory.
```

Comment (JohanFreelancer9) [reply microsoft-365/solutions/collaborate-as-team.md:52]: Please remove the extra space between "as" and "regular".

File: microsoft-365/solutions/collaborate-as-team.md [hunk 5]
```diff
@@ -29,25 +29,34 @@ This video shows the configuration steps described in this document.</br>
...
+   - **Admins and users in the guest inviter role can invite**: To allow admins and users in the "Guest Inviter" role to invite guests, set this policy to **Yes**.
+   - **Members can invite**: To allow non-admin members of your directory to invite guests, set this policy to **Yes**.
+   - **Guests can invite**: To allow guests to invite other guests, set this policy to **Yes**.
+   - **Enable Email One-Time Passcode for guests (Preview)**: For more information about the one-time passcode feature, see [Email one-time passcode authentication (preview)](https://docs.microsoft.com/en-us/azure/active-directory/b2b/one-time-passcode).
```
Comment (JohanFreelancer9) [inline microsoft-365/solutions/collaborate-as-team.md]:
```suggestion
   - **Enable Email One-Time Passcode for guests (Preview)**: For more information about the one-time passcode feature, see [Email one-time passcode authentication (preview)](https://docs.microsoft.com/azure/active-directory/b2b/one-time-passcode).
```

File: microsoft-365/solutions/collaborate-as-team.md [hunk 6]
```diff
@@ -29,25 +29,34 @@ This video shows the configuration steps described in this document.</br>
...
+   - **Collaboration restrictions**: For more information about allowing or blocking invitations to specific domains, see [Allow or block invitations to B2B users from specific organizations](https://docs.microsoft.com/en-us/azure/active-directory/b2b/allow-deny-list).
```
Comment (JohanFreelancer9) [inline microsoft-365/solutions/collaborate-as-team.md]:
```suggestion
   - **Collaboration restrictions**: For more information about allowing or blocking invitations to specific domains, see [Allow or block invitations to B2B users from specific organizations](https://docs.microsoft.com/azure/active-directory/b2b/allow-deny-list).
```

File: microsoft-365/solutions/collaborate-as-team.md [hunk 7]
```diff
@@ -29,25 +29,34 @@ This video shows the configuration steps described in this document.</br>
...
+   - **Enable guest self-service sign up via user flows (Preview)**: For more information about the Guest self service sign up via user flows, see [self service sign up user flows (Preview)](https://docs.microsoft.com/en-us/azure/active-directory/b2b/self-service-sign-up-user-flow).
```
Comment (JohanFreelancer9) [inline microsoft-365/solutions/collaborate-as-team.md]:
```suggestion
   - **Enable guest self-service sign up via user flows (Preview)**: For more information about guest self-service sign up via user flows, see [Add a self-service sign-up user flow to an app (Preview)](https://docs.microsoft.com/azure/active-directory/b2b/self-service-sign-up-user-flow). 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by PrasadKasireddy** [comment]
> Copy review.

@JohanFreelancer9  - thanks for your inputs- I updated the changes as per your comments

**Comment by mypil** [comment]
@MikePlumleyMSFT - PR has been copy edited and is ready for final review. This is based on issue #2007.


Thank you.

cc: @kenwith

**Comment by MikePlumleyMSFT** [comment]
Primary nav should be via External Identities > External collaboration settings and screenshot should show this as in original. Also, extra details about each setting not needed. Updating via a separate PR.

**Pull request closed without merging**